packaging/dependencies.nix

```diff
@@ -65,6 +66,18 @@ scope: {
         installPhase = lib.replaceStrings [ "--without-python" ] [ "" ] old.installPhase;
       });
 
+  libblake3 = pkgs.libblake3.overrideAttrs (attrs: rec {
+    version = "1.7.0";
```

Comment:
Can you also bump this in nixpkgs?

Reply:
@Mic92 Yes, there is a pending PR for that here: https://github.com/NixOS/nixpkgs/pull/390458

Reply:
I merged it